src/test/java/org/apache/commons/lang3/StringUtilsContainsTest.java

```diff
@@ -37,54 +34,22 @@ public class StringUtilsContainsTest extends AbstractLangTest {
     public void testContains_Char() {
         assertFalse(StringUtils.contains(null, ' '));
         assertFalse(StringUtils.contains("", ' '));
-        assertFalse(StringUtils.contains("", null));
```

Comment:
@MSaifAsif 
Do NOT delete existing tests please.

Reply:
I migrated it over [here](https://github.com/apache/commons-lang/pull/1279/files#diff-26c06c4f25342a7e49477cabfc7198f12a7d76023e2a7f3987d2341619669eb6R33). Would you prefer that we keep the existing StringUtils test cases as well on top of the separation that I did ?